src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -653,15 +653,25 @@ export default class AircraftCommander {
             return [false, readback];
         }
 
+        // TODO: move somewhere else
+        const previousAircraft = this._aircraftController.findAircraftByCallsign(runway.lastAircraftTakenoff);
+        if (previousAircraft) {
+            const actualDistance = aircraft.positionModel.distanceToPosition(previousAircraft.positionModel) * 1000;
```

Comment:
This returns the distance in nautical miles, which are about 6000ft. You can use `nm_ft()` to convert and drop the `* 1000`.

Reply:
`nm_ft()` does not exist.